packages/components/Badge/src/__tests__/__snapshots__/Badge.test.tsx.snap

```diff
@@ -17,7 +17,7 @@ exports[`Badge component tests Badge all props 1`] = `
       "justifyContent": "center",
       "minHeight": 24,
       "minWidth": 24,
-      "paddingHorizontal": 8,
+      "paddingHorizontal": 4,
```

Comment:
Are these changes expected? The visuals as in snaps seem updated

Reply:
Let me check this

Reply:
I think ideally this PR should not have any changes to the snapshots, but one of the valid reasons why the snapshots might have changed in this PR is that when this other PR got introduced: https://github.com/microsoft/fluentui-react-native/pull/2242/files we switched from using windows token values to using iOS token values (which were outdated at the time), which changed a bunch of snapshots. I think some of the changes you've made are just reverting the values to what they used to be, which is what we want.

Reply:
Yes make sense.